lib/europeana/blacklight/search_helper.rb

```diff
@@ -0,0 +1,44 @@
+module Europeana
+  module Blacklight
+    ##
+    # Local overrides for {Blacklight::SearchHelper}
+    module SearchHelper
+      def previous_and_next_document_params(index, window = 1)
+        api_params = {}
+
+        if index > 1
+          api_params[:start] = index - window # get one before
+          api_params[:rows] = 2 * window + 1 # and one after
+        else
+          api_params[:start] = 1 # there is no previous doc
+          api_params[:rows] = 2 * window # but there should be one after
+        end
+
+        api_params
+      end
+
+      private
+
+      def fetch_many(ids = [], *args)
+        if args.length == 1
+          user_params = params
+          extra_controller_params = args.first || {}
+        else
+          user_params, extra_controller_params = args
+          user_params ||= params
+          extra_controller_params ||= {}
+        end
+
+        id_query = ids.map { |id| "europeana_id:\"/#{id}\"" }.join(' OR ')
+
+        query = search_builder.
+                  with(user_params).
+                  where(id_query).
```

Comment:
Use 2 (not 10) spaces for indenting an expression in an assignment spanning multiple lines.